Source/BugsnagErrorTypes.h

```diff
@@ -0,0 +1,49 @@
+//
+//  BugsnagErrorTypes.h
+//  Bugsnag
+//
+//  Created by Jamie Lynch on 22/04/2020.
+//  Copyright © 2020 Bugsnag. All rights reserved.
+//
+
+#import <Foundation/Foundation.h>
+
+@interface BugsnagErrorTypes : NSObject
+
+/**
+ * Determines whether Out of Memory events should be reported to bugsnag.
+ *
+ * This flag is true by default.
+ */
+@property BOOL OOMs;
```

Comment:
There's an unfortunate set of casing in these fields! I can see why you've chosen these, but is there anything we can do to improve the naming?

Reply:
Agreed to rename as the following for now:

- ooms
- unhandledExceptions
- signals
- cppExceptions
- machExceptions